### ISSUE TITLE
Glossed examples in bulleted list display with a blank line to the right of the bullet in Chrome

### ISSUE THREAD

**Issue by nschneid** [nert-gu/Xposition#141]
Apparently there is an empty paragraph within the `<li>` that Firefox does not display but Chrome displays as a blank line.

**Comment by nschneid** [comment]
In fact, there are several empty paragraphs: one containing an empty span, one with `class="gloss"`, and one before the translation.

```
<li>
<p><span class="example" id="001">
                    </span></p><div class="interlinear example">
                    <p class="gloss">
                        </p><div class="gll">L'<br>the</div><div class="gll">éléphant<br>elephant</div><div class="gll">gris<br>gray</div><div class="gll">est<br></div><div class="gll"><a class="usage" href="/fr/dans/Locus--Locus">dans</a><br><a class="usage" href="/en/in/Locus--Locus">in</a></div><div class="gll">la voiture<br>the car</div><div class="gll">.</div>
                    <p></p>
                    <p class="translation">The gray elephant is in the car.&nbsp;<a class="exlabel" href="#001">001</a></p>
                    </div>
                    <p></p>
</li>
```

If it's difficult to modify the HTML we may be able to use `display: none` CSS on some of them.

**Comment by nschneid** [comment]
Fixed in 8fc1ee0. The macro was generating `<span class="example" id="001"><div class="interlinear example">...`, but `<div>` (a block-level element) cannot be nested within `<span>` (an inline-level element), so the browser was putting the empty span in a separate paragraph.
